src/t_stream.c

```diff
@@ -1259,6 +1248,13 @@ void xaddCommand(client *c) {
             break;
         }
     }
+
+    /* Check for no_mkstream option */
+    if (c->argc-i > 1 && !strcasecmp(c->argv[i+1]->ptr,"nomkstream")) {
+        no_mkstream = 1;
+        i++;
+    }
+
```

Comment:
i think i'd rather have this optional argument next to the other optional argument(s) (MAXLEN).
These optional arguments should be supported at any order IMHO.

Reply:
thanks @oranagra , will fix.